lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -655,6 +655,34 @@ class ConvertTritonGPUOpToLLVMPatternBase {
     llvm_unreachable("unsupported emitOffsetForLayout");
   }
 
+#ifdef USE_ROCM
+  void emitMfmaOffsetForCTA(const MfmaEncodingAttr &mfmaLayout,
+                            SmallVector<SmallVector<unsigned>> &offsets,
+                            unsigned ctaOffsetX, unsigned ctaOffsetY) const {
+    // MFMA output tile consists of repeated "dot operand B" layout blocks along
+    // row axis. This variable defines number of these blocks
+    const unsigned numBlocks = 4;
+    const unsigned elemsPerThreadPerBlock = 4;
+    auto warpSize = getWarpSize(mfmaLayout);
+    assert(warpSize == 64);
+    auto shapePerCta = getShapePerCTA(mfmaLayout);
+    for (unsigned block = 0; block < numBlocks; block++) {
+      unsigned rowOrColOffset = block * elemsPerThreadPerBlock * warpSize / 32;
+      for (unsigned elem = 0; elem < elemsPerThreadPerBlock; elem++) {
+        if (mfmaLayout.getIsTransposed()) {
```

Comment:
I guess this is where the conversion(#mfma.trans, #blocked) is handled when storing the second gemm's results to global memory?